src/Catalyst.Core.Lib/DAO/Transaction/PublicEntryDao.cs

```diff
@@ -62,16 +63,18 @@ public void InitMappers(IMapperConfigurationExpression cfg)
             cfg.AllowNullDestinationValues = true;
 
             cfg.CreateMap<PublicEntry, PublicEntryDao>()
-               .ForMember(d => d.Id, opt => opt.MapFrom(src => _hashProvider.ComputeMultiHash(src.ToByteArray())))
+               .ForMember(d => d.Id, opt => opt.MapFrom(src => _hashProvider.ComputeMultiHash(src)))
                .ForMember(d => d.Amount,
                     opt => opt.ConvertUsing(new ByteStringToUInt256StringConverter(), s => s.Amount))
                .ForMember(e => e.Data, opt => opt.ConvertUsing<ByteStringToBase32Converter, ByteString>())
                .ForMember(d => d.ReceiverAddress,
                     opt => opt.ConvertUsing(new ByteStringToBase32Converter(), s => s.ReceiverAddress))
                .ForMember(d => d.SenderAddress,
                     opt => opt.ConvertUsing(new ByteStringToBase32Converter(), s => s.SenderAddress))
-               .ForMember(d => d.TransactionFees,
-                    opt => opt.ConvertUsing(new ByteStringToUInt256StringConverter(), s => s.TransactionFees));
+               .ForMember(d => d.Nonce, opt => opt.MapFrom(s => s.Nonce))
```

Comment:
we ditched transaction fees - this is something we need to discuss in details - in EVM model the actual price is only known after the execution

Reply:
Created a ticket for this https://github.com/catalyst-network/Catalyst/issues/1168